ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,278 @@
+        mv /etc/elasticsearch/certs/filebeat.key /etc/filebeat/certs/ 
+        mv /etc/elasticsearch/certs/filebeat.pem /etc/filebeat/certs/ 
+        # Start Filebeat
+        startService "filebeat"
+        logger "Done"
+    fi
+}
+
+## Kibana
+installKibana() {
+    WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
+    logger "Installing Open Distro for Kibana..."
+    $sys_type install opendistroforelasticsearch-kibana-${OPENDISTRO_VERSION} -y -q $debug
+    if [ "$?" != 0 ]; then
+        echo "Error: Kibana installation failed"
+        exit 1;
+    else
+        curl -so /etc/kibana/kibana.yml https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}/resources/open-distro/kibana/7.x/kibana_all_in_one.yml --max-time 300 $debug
+        echo "telemetry.enabled: false" >> /etc/kibana/kibana.yml
+        cd /usr/share/kibana $debug
+
+        if [ "${STATUS_PACKAGES}" = "prod" ]; then
+            sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages.wazuh.com/${WAZUH_MAJOR}.x/ui/kibana/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}.zip $debug
+        elif [ "${STATUS_PACKAGES}" = "dev" ]; then
+            sudo -u kibana /usr/share/kibana/bin/kibana-plugin install https://packages-dev.wazuh.com/pre-release/ui/kibana/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}-1.zip $debug
+        fi
+                
+        if [ "$?" != 0 ]
+        then
+            echo "Error: Wazuh Kibana plugin could not be installed."
+            exit 1;
+        fi
+        mkdir -p /etc/kibana/certs $debug
+        mv /etc/elasticsearch/certs/kibana.key /etc/kibana/certs/ $debug
+        mv /etc/elasticsearch/certs/kibana.pem /etc/kibana/certs/ $debug
+        setcap 'cap_net_bind_service=+ep' /usr/share/kibana/node/bin/node $debug
+
+        # Start Kibana
+        startService "kibana"
+
+        logger "Done"
+    fi
+}
+
+## Health check
+healthCheck() {
+
+    cores=$(nproc)
+    ram_gb=$(free -m | awk '/^Mem:/{print $2}')
+
+    if [[ $cores -lt 2 ]] || [[ $ram_gb -lt 4096 ]]; then
+        echo "The system must have at least 4Gb of RAM and 2 CPUs"
+        exit 1;
+    else
+        echo "Starting the installation..."
+    fi
+}
+
+checkInstallation() {
+
+    logger "Checking the installation..."
+    curl -XGET https://localhost:9200 -uadmin:admin -k --max-time 300 $debug
+    if [ "$?" != 0 ]; then
+        echo "Error: Elasticsearch was not successfully installed."
+        exit 1
+    else
+        echo "Elasticsearch installation succeeded."
+    fi
+    filebeat test output $debug
+    if [ "$?" != 0 ]; then
+        echo "Error: Filebeat was not successfully installed."
+        exit 1;
+    else
+        echo "Filebeat installation succeeded."
+    fi
+    logger "Initializing Kibana (this may take a while)"
+    until [[ "$(curl -XGET https://localhost/status -I -uadmin:admin -k -s | grep "200 OK")" ]]; do
+        echo -ne $char
+        sleep 10
+    done
+    echo $'\nInstallation finished'
```

Comment:
Use `logger` function.

Reply:
Done in 53208f3d9e390ee69e4f23799af5b3009001d9c2